src/api/main/credrank.js

```diff
@@ -44,13 +51,6 @@ export async function credrank(input: CredrankInput): Promise<CredrankOutput> {
     merge(input.pluginGraphs),
     input.weightOverrides
   );
-  const dependenciesWithIds = input.dependencies.map((d) =>
-    // This mutates the ledger, adding new identities when needed.
-    ensureIdentityExists(d, input.ledger)
-  );
-  const dependencyBonuses = dependenciesWithIds.map((d) =>
-    toBonusPolicy(d, input.ledger)
-  );
```

Comment:
why move this in this commit? what does this have to do with the attribution config?

Reply:
Just baking in a little organization of the method by putting all the dependency-related code together.